ctags/main/main.c

```diff
@@ -46,7 +46,7 @@
 #  include <sys/types.h>  /* required by dirent.h */
 # endif
 # include <dirent.h>  /* to declare opendir() */
-# undef boolean
+# undef bool
```

Comment:
Crap, yeah, I should have gone through all the uses manually instead of thinking simple rename will do the right thing. Will go through all the renames and fix issues.